frontend/src/assets/main.css

```diff
@@ -478,3 +478,16 @@ h1 {
 		max-height: 70vh;
 	}
 }
+
+.link_button {
+	-webkit-appearance: none;
+	appearance: none;
+	background: none;
+	border: 0px;
+	padding: 0;
+	font-style: italic;
```

Comment:
could you add this as well?
```css
cursor: pointer;
```

Reply:
I had it, then removed, as it seemed not to make a difference, maybe just very slow hot reloading on my ancient machine :D